CMakeLists.txt

```diff
@@ -184,8 +184,14 @@ endif()
 option(SFML_ENABLE_STDLIB_ASSERTIONS "Enable standard library assertions" OFF)
 if(SFML_ENABLE_STDLIB_ASSERTIONS)
     # intentionally using `add_definitions` here to propagate defines to subdirectories
```

Comment:
What about this comment about intentionally using `add_definitions`? Can we safely change to `add_compile_definitions`?

Reply:
Yeah it's safe, they both set the directory on the property which will propagate to subdirectories, but I'll update the comment or remove it if preferred as I think the intention is quite clear already